mappings/markdown/Prowler/samples/prowler.ocsf

```diff
@@ -0,0 +1,143 @@
+{
+    "metadata": {
+        "event_code": "acm_certificates_expiration_check",
+        "product": {
+            "name": "Prowler",
+            "vendor_name": "Prowler",
+            "version": "4.4.0"
+        },
+        "version": "1.2.0"
+    },
+    "severity_id": 4,
+    "severity": "High",
+    "status": "Other",
```

Comment:
Note, if your product doesn't support or populate status values of a finding, I would recommend not populating the field instead of populating it as `Other`